File: packages/csharp/src/Errors/TransactionOrderNotFoundException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>404 TRANSACTION_ORDER_NOT_FOUND.</summary>
public sealed class TransactionOrderNotFoundException : NotFoundException
{
    /// <summary>Construct with the full required-field set.</summary>
    public TransactionOrderNotFoundException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}

File: packages/java/src/main/java/com/tesote/sdk/errors/TransactionOrderNotFoundException.java
package com.tesote.sdk.errors;

public final class TransactionOrderNotFoundException extends NotFoundException {
    public TransactionOrderNotFoundException(String message, String errorCode, int httpStatus,
                                             String requestId, String errorId, Integer retryAfter,
                                             String responseBody, RequestSummary requestSummary,
                                             int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}
